### PR TITLE
add postgres support

### PR DESCRIPTION
**Pull request by sutidor**
successfully tested with 

FF_SYNCSERVER_SQLURI=postgresql://firefox-syncserver_db_user:PASSWORD@postgres/firefox-syncserver_db

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,17 +21,22 @@ RUN apk --update --no-cache add \
     shadow \
     su-exec \
     tzdata \
+    postgresql-dev \
```
Comment (crazy-max) [inline Dockerfile:24]: Should be placed in build-dependencies and replace with `postgresql-client` instead no?

File: Dockerfile [hunk 2]
```diff
@@ -21,17 +21,22 @@ RUN apk --update --no-cache add \
...
   && apk --update --no-cache add -t build-dependencies \
     build-base \
     git \
     libffi-dev \
     libressl-dev \
+    gcc \
+    python-dev \
+    musl-dev \
```
Comment (crazy-max) [inline Dockerfile:32]: Please place packages in alphabetical order.

File: Dockerfile [hunk 3]
```diff
@@ -21,17 +21,22 @@ RUN apk --update --no-cache add \
...
   && git clone https://github.com/mozilla-services/syncserver app \
   && cd app \
   && git reset --hard $SHA1_COMMIT \
   && pip install pymysql \
   && pip install --upgrade --no-cache-dir -r requirements.txt \
   && pip install --upgrade --no-cache-dir -r dev-requirements.txt \
+  && pip install psycopg2 \
```
Comment (crazy-max) [inline Dockerfile:39]: Instead of adding a new line you can replace `&& pip install pymysql \`
with `&& pip install pymysql psycopg2 \` ?

**Review by crazy-max — CHANGES_REQUESTED** [review]
Thanks for this PR! Some changes you can make and also add a [note in the README about PostgreSQL support](https://github.com/crazy-max/docker-firefox-syncserver#notes)

File: Dockerfile (hunk 1 above)
Comment (sutidor) [reply Dockerfile]: I tried that, but it did not work for build

**Review by crazy-max — APPROVED** [review]
LGTM thanks

**Comment by sutidor** [comment]
> 
> 
> LGTM thanks

& works like a charm. Thanks for maintaining this great repo

**Pull request merged**